src/pages/Dashboard.tsx

```diff
@@ -122,6 +172,38 @@ const Dashboard = () => {
             <Button onClick={createNewInsight}>
                 New Insight
             </Button>
+            {/**TODO: Popup form for new group**/}
```

Comment:
If you want a popup form, you can user MUI's `<Modal />` and then you won't have to user zIndexes in styles